### ISSUE TITLE
Off-road route

### ISSUE THREAD

**Issue by SovS-UA** [osmandapp/Osmand#4898]
Hello, I can not make the navigation application OsmAnd on Android go on the road through Karshenno Annenka and Grekovo-Ulyanovka. Route from Matveyev Kurgan to Rodionovo-Nesvetayskaya. https://www.openstreetmap.org/directions?engine=osrm_car&route=47.5747%2C38.8834%3B47.6163%2C39.7070#map=12/47.5786/39.2195&layers=N

**Comment by xmd5a2** [comment]
It is possible if you add transit point

**Comment by SovS-UA** [comment]
Yes it is certainly possible, but if I'm going along the asphalt road through Karshenno-Annenka, then the application will tell me to turn around until I reach the Bolshekrepinskaya. Why does not the application offer an alternative way?
